openwisp_monitoring/monitoring/base/models.py

```diff
@@ -39,12 +40,14 @@
 
 
 class AbstractMetric(TimeStampedEditableModel):
+    METRICS = get_metric_configuration()
     name = models.CharField(max_length=64)
     description = models.TextField(blank=True)
```

Comment:
take advantage of this to remove this field and close https://github.com/openwisp/openwisp-monitoring/issues/28